Automattic/jetpack
# Slideshow Block: Add a bit of CSS to support align wide and align full

Fixes #12178

## Proposed changes

Adds a bit of Slideshow Block CSS to support align wide and align full controls:

![image](https://user-images.githubusercontent.com/36432/179253079-fc66ac31-be8d-4689-84a3-d717b7a8661f.png)

I personally tested in Firefox 102, Chrome 103, and Safari 15.5 with the following themes:
* Twenty Nineteen
* Twenty Sixteen
* Twenty Twenty
* Twenty Twenty-One
* Twenty Twenty-Two
* Blockbase
* Hemingway Rewritten

### Align Full Before/After

**Before**:

<img width="1768" alt="image" src="https://user-images.githubusercontent.com/36432/179246755-0be984d8-5b85-44e3-9d0b-535929715885.png">

**After**:

<img width="1767" alt="image" src="https://user-images.githubusercontent.com/36432/179246435-8640052a-a581-42cb-99db-9595083c393b.png">

### Align Wide Before/After

**Before**:

<img width="1766" alt="image" src="https://user-images.githubusercontent.com/36432/179246680-85fe265d-25d7-4992-8faf-bea4430a9e0b.png">

**After**:

<img width="1769" alt="image" src="https://user-images.githubusercontent.com/36432/179246498-9d8629de-eda0-4950-a7c1-9bd01c19e139.png">

## Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

Some prior discussion about testing in p1657651639634659-slack-CBG1CP4EN

1. Check out the branch in your preferred testing environment. 
2. Create a new post with a Slideshow Block, Cover Block, and Gallery Block, all set to "Align None". Open in a new window.
3. Copy all of the blocks into a second post, set each block to "Align Wide", and open in a new window.
4. Copy all of the blocks into a third post, set each block to "Align Full", and open in a new window.
5. View each of your three posts and verify the Slideshow Block follows the same behavior as the Cover Block and Gallery Block.
6. Switch between a few themes and verify each block displays consistent behavior in each theme.
